megamek/src/megamek/common/alphaStrike/ASSpecialAbilityCollection.java

```diff
@@ -49,6 +49,10 @@ public boolean isEmpty() {
         return specialAbilities.isEmpty();
     }
 
+    public EnumMap<BattleForceSUA, Object> getInternalRepr() {
+        return specialAbilities;
```

Comment:
exposing the internal representation of the SUA because I need it to find out what are the damage ranges of warships